src/App.js

```diff
@@ -1,8 +1,16 @@
 import MockmanEs from "mockman-js";
-import { Route, Routes } from "react-router-dom";
+import React, { useEffect } from "react";
+import { Route, Routes, useNavigate } from "react-router-dom";
 import "./App.css";
 import { Footer, Navbar } from "./components";
-import { LandingPage, LoginPage, SignupPage, NotesPage } from "./pages/index";
+import {
+  LandingPage,
+  LoginPage,
+  SignupPage,
+  NotesPage,
+  ArchivePage,
+} from "./pages/index";
+const encodedToken = localStorage.getItem("Mananote.Token");
```

Comment:
I am not getting it, why are you getting token in your App.js take it out only when necesaary and where it is required.